src/network/webtask.h

```diff
@@ -150,6 +150,8 @@ class WebTask : public NetworkTask {
         Idle,
         // Pending state
         Pending,
+        // Looping state
+        Looping,
```

Comment:
```suggestion
        Waiting,
```
This is the state where the task is waiting until the time for rate limiting is over. 
Or WaitingForNextRequestSlot.

Reply:
Thank you for the review. Done :+1: